### PR TITLE
WD-9529 - resize image on /data/opensearch

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Make logo on /data/opensearch larger

## QA

- [demo link](https://canonical-com-1203.demos.haus/data/opensearch)
- [linked issue](https://warthogs.atlassian.net/browse/DPNPM-415)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the logo is larger

## Issue / Card
[WD-9529](https://warthogs.atlassian.net/browse/WD-9529)
Fixes #

## Screenshots

[WD-9529]: https://warthogs.atlassian.net/browse/WD-9529?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lizzochek** [comment]
@lorumic I confirmed it with @Sophie-32 Not sure whether it needs someone else to confirm

**Comment by akbarkz** [comment]
@lizzochek I think it only needs the stakeholder's approval now. I sent her a link to demo instance, let's wait for her confirmation.

**Review by akbarkz — CHANGES_REQUESTED** [review]
Please make the size of the logo similar to one on MySQL data page. Also pls fix the size for Kafka and Spark pages accordingly.

**Review by akbarkz — APPROVED** [review]

**Comment by akbarkz** [comment]
duplicate of https://github.com/canonical/canonical.com/pull/1205

**Pull request closed without merging**
